LATEST

```diff
@@ -1,4 +1,4 @@
-021f4af394ceb34d9cdfb64e6ac9b908ca2da0f8 1.2.0-snapshot.20200513.4172.0.021f4af3
+021f4af394ceb34d9cdfb64e6ac9b908ca2da0f8 1.2.0-snapshot.20200520.4224.0.2af134ca
```

Comment:
why are you not bumping the commit?

Reply:
Haven't had my coffee yet I guess. Also, CI caught it! 🎉